courses/utils.py

```diff
@@ -75,3 +79,46 @@ def process_course_run_grade_certificate(course_run_grade):
             user=user, course_run=course_run
         )
         return certificate, created, False
+
+
+def generate_program_certificate(user, program):
+    """
+    Create a program certificate if the user has a course certificate
+    for each course in the program
+
+    Args:
+        user (User): a Django user.
+        program (programs.models.Program): program where the user is enrolled.
+
+    Returns: (ProgramCertificate or None, bool): A tuple containing a
```

Comment:
Please add line breaks and indents like the rest of our docstrings ([example](https://github.com/mitodl/mitxpro/blob/master/courses/api.py#L27-L28))